test

```diff
@@ -39,8 +39,7 @@ else
     fi
 fi
 
-if [[ "$run_jshint" == "true" ]]; then
-    jshint src \
-        && jshint --config tests/.jshintrc tests \
-        && echo "jshint passed"
+if [[ "$run_lint" == "true" ]]; then
+    npx eslint .
+        && echo "linting passed"
```

Comment:
I think this should either be changed to a single line or a line continuation character is needed.
```suggestion
    npx eslint . && echo "linting passed"
```